lib/ansible/modules/cloud/amazon/iam_cert.py

```diff
@@ -97,16 +80,22 @@
 '''
 
 EXAMPLES = '''
-# Basic server certificate upload
-tasks:
-- name: Upload Certificate
-  iam_cert:
+# Basic server certificate upload from local file
+- iam_cert:
     name: very_ssl
     state: present
-    cert: somecert.pem
-    key: privcertkey
-    cert_chain: myverytrustedchain
+    cert: "{{ lookup('file', 'path/to/cert') }}"
```

Comment:
@ProjectRice Good point. If you get a chance would you please review https://github.com/ansible/ansible/pull/26097 so paths can be specified?